### DISCUSSION TITLE
Why can't I see watched repository activity in my feed?

### DISCUSSION THREAD

**Discussion by chakravala** [community/community#109290] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The feed used to tell me about repositories I am watching, now it doesn't.

I'm not even getting all the emails about the commits of the repositories I am watching.

Something is very wrong, seems GitHub is going down the drain and turning into useless garbage.

**Comment by NineIT420** [comment]
I understand your frustration with the changes to your GitHub notifications. It seems like some updates or modifications have affected the repositories you are watching, making it more challenging to follow their progress.
Initially, what we can do is to check the settings carefully to ensure the notification preferences are correctly configured. If everything is adjusted correctly and you're still experiencing issues, we could contact GitHub support to investigate the problem further.
This situation might also indicate a wider trend of GitHub not meeting user expectations.
If this continues, consider exploring alternatives like Bitbucket or GitLab, which might offer the features you require.
But before making any drastic changes, it's best to ensure the issue isn't a minor glitch that can be fixed conveniently.

Let's try to rectify this issue and continue using the platform you're familiar with.
